src/db.ts

```diff
@@ -821,6 +834,7 @@ export abstract class Database {
         !(r.inv in crit),
         `${r.runid}, ${r.criterion}, ${r.inv} in ${JSON.stringify(crit)}`
       );
+      // still needed?
```

Comment:
I don't know what the code is doing, but I suspect yes.
This is using `fetchMaxMeasurements` which I suggested a fix for.